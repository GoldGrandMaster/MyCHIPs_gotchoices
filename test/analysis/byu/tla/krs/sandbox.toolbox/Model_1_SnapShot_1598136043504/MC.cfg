\* SPECIFICATION definition
SPECIFICATION
Spec
\* INVARIANT definition
INVARIANT
MoneyNotNegative
MoneyInvariant
\* Generated on Sat Aug 22 16:40:39 MDT 2020