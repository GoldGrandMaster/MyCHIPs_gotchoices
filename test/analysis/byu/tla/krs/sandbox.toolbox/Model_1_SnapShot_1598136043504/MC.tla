---- MODULE MC ----
EXTENDS sandbox, TLC

=============================================================================
\* Modification History
\* Created Sat Aug 22 16:40:39 MDT 2020 by kylestorey
