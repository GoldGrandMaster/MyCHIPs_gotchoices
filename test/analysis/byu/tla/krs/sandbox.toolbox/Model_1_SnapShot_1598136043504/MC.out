@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.15 of Day Month 20?? (rev: eb3ff99)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2187:0 @!@!@
Running breadth-first search Model-Checking with fp 115 and seed 5158218642252310074 with 2 workers on 4 cores with 608MB heap and 1365MB offheap memory [pid: 64707] (Mac OS X 10.13.6 x86_64, AdoptOpenJDK 14.0.1 x86_64, OffHeapDiskFPSet, DiskStateQueue).
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file /Users/kylestorey/workspace/MyCHIPs/test/analysis/krs/sandbox.toolbox/Model_1/MC.tla
Parsing file /Users/kylestorey/workspace/MyCHIPs/test/analysis/krs/sandbox.toolbox/Model_1/sandbox.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/TLC.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Naturals.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/Sequences.tla
Parsing file /Applications/TLA+ Toolbox.app/Contents/Eclipse/plugins/org.lamport.tlatools_1.0.0.202004251858/tla2sany/StandardModules/FiniteSets.tla
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module FiniteSets
Semantic processing of module TLC
Semantic processing of module sandbox
Semantic processing of module MC
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2020-08-22 16:40:40)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 1000:1 @!@!@
@!@!@STARTMSG 2266:0 @!@!@
The first argument of > should be an integer, but instead it is:
<<1, 1>>
@!@!@ENDMSG 2266 @!@!@
@!@!@ENDMSG 1000 @!@!@
@!@!@STARTMSG 2103:1 @!@!@
The error occurred when TLC was evaluating the nested
expressions at the following positions:
0. Line 29, column 9 to line 34, column 50 in sandbox
1. Line 29, column 12 to line 29, column 29 in sandbox
2. Line 30, column 12 to line 30, column 27 in sandbox
3. Line 31, column 12 to line 31, column 54 in sandbox
4. Line 33, column 12 to line 33, column 36 in sandbox
5. Line 34, column 12 to line 34, column 50 in sandbox
6. Line 70, column 21 to line 70, column 30 in sandbox


@!@!@ENDMSG 2103 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics at 2020-08-22 16:40:41
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2773:0 @!@!@
<Init line 28, col 1 to line 28, col 4 of module sandbox>: 2:2
@!@!@ENDMSG 2773 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 29, col 9 to line 34, col 50 of module sandbox: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Transfer_ line 36, col 1 to line 36, col 15 of module sandbox>: 0:0
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 38, col 34 to line 38, col 64 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 39, col 34 to line 39, col 64 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 40, col 23 to line 41, col 43 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<A line 43, col 1 to line 43, col 7 of module sandbox>: 0:0
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 44, col 15 to line 44, col 58 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 45, col 15 to line 45, col 54 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 46, col 15 to line 46, col 45 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 47, col 15 to line 47, col 50 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<C line 49, col 1 to line 49, col 7 of module sandbox>: 0:0
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 52, col 15 to line 52, col 48 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 53, col 15 to line 53, col 78 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Terminating line 58, col 1 to line 58, col 11 of module sandbox>: 0:0
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 59, col 19 to line 59, col 32 of module sandbox: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2774:0 @!@!@
<MoneyNotNegative line 70, col 1 to line 70, col 16 of module sandbox>
@!@!@ENDMSG 2774 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 70, col 21 to line 70, col 30 of module sandbox: 2
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2774:0 @!@!@
<MoneyInvariant line 71, col 1 to line 71, col 14 of module sandbox>
@!@!@ENDMSG 2774 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(1) at 2020-08-22 16:40:42: 1 states generated (24 s/min), 1 distinct states found (24 ds/min), 1 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
1 states generated, 1 distinct states found, 1 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 2434ms at (2020-08-22 16:40:42)
@!@!@ENDMSG 2186 @!@!@
